projects/1.18/assets/metallum/metallum/lang/zh_cn.json

```
@@ -0,0 +1,52 @@
+{
+  "block.metallum.alchemical_bricks": "炼金砖",
+  "block.metallum.alchemical_tiles": "炼金瓦",
+  "block.metallum.cracked_alchemical_bricks": "裂纹炼金砖",
+  "block.metallum.cracked_alchemical_tiles": "裂纹炼金瓦",
+  "block.metallum.chiseled_alchemical_tile": "錾制炼金瓦",
+  "block.metallum.alchemical_brick_stairs": "炼金砖楼梯",
+  "block.metallum.alchemical_tiles_stairs": "炼金瓦楼梯",
+  "block.metallum.alchemical_brick_slab": "炼金砖台阶",
+  "block.metallum.alchemical_tiles_slab": "炼金瓦台阶",
+
+  "item.metallum.iron_cluster_sword": "铁集簇剑",
+  "item.metallum.iron_cluster_pickaxe": "铁集簇镐",
+  "item.metallum.iron_cluster_axe": "铁集簇斧",
+  "item.metallum.iron_cluster_shovel": "铁集簇锹",
+  "item.metallum.iron_cluster_hoe": "铁集簇锄",
+  "item.metallum.golden_cluster_sword": "金集簇剑",
+  "item.metallum.golden_cluster_pickaxe": "金集簇镐",
+  "item.metallum.golden_cluster_axe": "金集簇斧",
+  "item.metallum.golden_cluster_shovel": "金集簇锹",
+  "item.metallum.golden_cluster_hoe": "金集簇锄",
+  "item.metallum.copper_cluster_sword": "铜集簇剑",
+  "item.metallum.copper_cluster_pickaxe": "铜集簇镐",
+  "item.metallum.copper_cluster_axe": "铜集簇斧",
+  "item.metallum.copper_cluster_shovel": "铜集簇锹",
+  "item.metallum.copper_cluster_hoe": "铜集簇锄",
+  "item.metallum.copper_sword": "铜剑",
+  "item.metallum.copper_pickaxe": "铜镐",
+  "item.metallum.copper_axe": "铜斧",
+  "item.metallum.copper_shovel": "铜锹",
+  "item.metallum.copper_hoe": "铜锄",
+  "item.metallum.alchemical_node": "炼金结点",
+  "item.metallum.alchemical_dust": "炼金粉",
+  "item.metallum.pinch_of_alchemical_dust": "小撮炼金粉",
+  "item.metallum.reagent_pendant": "触媒吊坠",
+
+  "malum.gui.book.entry.cluster_tools": "集簇工具",
+  "malum.gui.book.entry.cluster_tools.description": "与大地调谐",
+  "malum.gui.book.entry.reagent_pendant": "触媒吊坠",
+  "malum.gui.book.entry.reagent_pendant.description": "秘术把戏",
+  "malum.gui.book.entry.alchemical_blocks": "炼金方块",
+  "malum.gui.book.entry.alchemical_blocks.description": "非自然石头",
+
+  "malum.gui.book.entry.page.headline.cluster_tools": "集簇工具",
+  "malum.gui.book.entry.page.text.cluster_tools_a": "将工具和对应的金属结点调谐似 乎能给工具带来一些奇妙的效果。 在工具内流动的炼金能量不仅让 其更经久耐用，也改变了它们的 外形。这些工具在挖掘和战斗中 都非常有用。",
+  "malum.gui.book.entry.page.headline.reagent_pendant": "触媒吊坠",
+  "malum.gui.book.entry.page.text.reagent_pendant_a": "将炼金粉和咒灰混合就能制成一 种反应活性极高的混合物。它能 和灵神剧烈反应，但可以通过镀 金吊坠稳定存放。在这种状态下， 它只会在接触外界灵神时激活， 并用澄空灵神的能量提供速度增 益。",
+  "malum.gui.book.entry.page.headline.alchemical_blocks": "炼金方块",
+  "malum.gui.book.entry.page.text.alchemical_blocks_a": "炼金粉可被压缩为炼金砖。裂纹、 瓦形、錾制、台阶、楼梯等等的 合成方式与岩石类似。",

```

```suggestion
  "malum.gui.book.entry.page.text.alchemical_blocks_a": "炼金粉可被压缩为炼金砖。炼金砖和石砖类似，可以用同样的方式合成为裂纹、瓦、錾制、台阶、楼梯等不同的形式。",
```
`stone counterparts`是炼金砖对应的石头物品，那就是石砖了。